message_sender/tasks.py

```diff
@@ -253,10 +259,12 @@ def run(self, message_id, **kwargs):
                 l.info('Connection Error sending message')
                 fire_metric.delay(
                     'sender.send_message.connection_error.sum', 1)
+                self.error_retry_count += 1
```

Comment:
Assigning to a class variable like `error_retry_count` doesn't increment the class variable but creates a new instance variable with the same name. Is that the behavior you intended here?